gem/templates/chhaajja/500.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html class="no-js">
+    <head>
+        <meta charset="utf-8" />
+        <title>Internal server error</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+    </head>
+    <body>
+        <h1>Internal server error</h1>
+
+        <h2>Sorry, there seems to be an error. Please try again soon.</h2>
```

Comment:
You should wrap static text with {% trans "static text here" %}  on the templates in order to generate a .po file Translations.